Library/ViewModels/DiscoveryPostcardViewModel.swift

```diff
@@ -76,6 +82,9 @@ public protocol DiscoveryPostcardViewModelOutputs {
   /// Emits a boolean to determine whether or not the metadata view should be hidden.
   var metadataViewHidden: Signal<Bool, NoError> { get }
 
+  /// Emits when we notify the delegate that the share button was tapped.
```

Comment:
the wording on this could be more `"Emits when we should notify the delegate..."`